### ISSUE TITLE
Add support and sensors for EquipmentData on v2 hub

### ISSUE THREAD

**Issue by msp1974** [asantaga/wiserHomeAssistantPlatform#512]

**Comment by msp1974** [comment]
Tagging @LGO44

**Comment by LGO44** [comment]
@msp1974 , hi Mark,
here is a file that presents the equipment data

[equipment data.pdf](https://github.com/user-attachments/files/17624262/equipment.data.pdf)

What are you expecting from me?

I saw that you have pubilshed a new version, do you want that I rebased all my modifications on it?

**Comment by dpgh947** [comment]
I don't understand what has just happened regarding this change. The latest integration update has included the changes in https://github.com/asantaga/wiserHomeAssistantPlatform/pull/651 . I am on a gen2 hub. I previously had "power" and "total power" entities on my wiser plugs, these are now gone and replaced by really weirdly named entities, two of which appear to be duplicated. For example, my plug named "lounge plug" now has these entities:
```
switch.wiser_lounge_plug_away_mode_turns_off
binary_sensor.wiser_lounge_plug_controllable
switch.wiser_lounge_plug_device_lock
switch.wiser_lounge_plug_identify
select.wiser_lounge_plug_mode
binary_sensor.wiser_lounge_plug_pcm_mode
sensor.wiser_lounge_plug_signal
switch.wiser_lounge_plug_switch
sensor.wiser_lounge_plug_wiser_device_4_equipment
sensor.wiser_lounge_plug_wiser_device_4_current
sensor.wiser_lounge_plug_wiser_device_4_equipment_energy_delivered
sensor.wiser_lounge_plug_wiser_device_4_equipment_power
sensor.wiser_lounge_plug_wiser_device_4_equipment_total_energy
```

I cannot work out why the "wiser_device_4" is inserted or where it is coming from. The device in HA is called "Wiser lounge plug", but if I use the "recreate entity IDs" option, it doesn't rename any.

Two are duplicated - "equipment" and "equipment_power" appear to be the same thing, what the old "power" entity showed, and similarly "equipment_energy_delivered" and "equipment_total_energy", what the old "total_power" showed - these even show the same values in all the example displays I found in the pull request and the pdf document linked above. This can't be right???

Bottom line - this change doesn't provide any new information for plugs as far as I can see, introduces an odd naming of the entities, and just causes work, with no warning about the changes.

**Comment by LGO44** [comment]
> I am on a gen2 hub. I previously had "power" and "total power" entities on my wiser plugs, these are now gone and replaced by really weirdly named entities, two of which appear to be duplicated. For example, my plug named "lounge plug" now has these entities:

The new entities are based on all the data provided by the hub V2.
The previous entities "power" and "total power"  could be reintroduced. 

> 
> I cannot work out why the "wiser_device_4" is inserted or where it is coming from. The device in HA is called "Wiser lounge plug", but if I use the "recreate entity IDs" option, it doesn't rename any.
> 
I 'll have a look on this name, and will make the name consistent with the previous naming

> Two are duplicated - "equipment" and "equipment_power" appear to be the same thing, what the old "power" entity showed, and similarly "equipment_energy_delivered" and "equipment_total_energy", what the old "total_power" showed - these even show the same values in all the example displays I found in the pull request and the pdf document linked above. This can't be right???
> 
You are right, I create all the entities based on the equipment data provided by the hub V2.

> Bottom line - this change doesn't provide any new information for plugs as far as I can see, introduces an odd naming of the entities, and just causes work, with no warning about the changes.

Sorry for all the work, and the trouble. In my case I never had the naming that 's why i make no warning about the changes.

The equipment provide the capability to send notifications  for under-power or over-power, and will be introduced in a next version

**Comment by asantaga** [comment]
@LGO44 can you take this and issue a PR? 
Ive closed this and created a new Issue , to keep things clean

**Comment by asantaga** [comment]
@dpgh947 @LGO44 opened https://github.com/asantaga/wiserHomeAssistantPlatform/issues/677

**Comment by LGO44** [comment]
> [@dpgh947](https://github.com/dpgh947) [@LGO44](https://github.com/LGO44) opened [#677](https://github.com/asantaga/wiserHomeAssistantPlatform/issues/677)

Ok for a PR
